davenjeru/authors-haven-playground
# Add the app via create-react-app

#### What does this PR do?
 - Adds the app via create-react-app

#### Description of Task to be completed?
 - Add the app via create-react-app

#### How should this be manually tested?
 - Clone this project with this [link](https://github.com/davenjeru/authors-haven-playground.git)
  `git clone https://github.com/davenjeru/authors-haven-playground.git`

 - Change the directory to **authors-haven-playground**
   `cd authors-haven-playground`

 - Checkout [this](https://github.com/davenjeru/authors-haven-playground/tree/create-react-app) branch
   `git checkout create-react-app`

 - Install the dependencies located in the [package.json](https://github.com/davenjeru/authors-haven-playground/tree/create-react-app/package.json)
   `npm install`

 - Start the project with a development server.
    `npm start`

 - Open the browser and you should see a page with the react logo

#### Any background context you want to provide?
 - The app has been built with React which uses [Node](https://nodejs.org/) and [npm](https://npmjs.org/) so they should be installed.

#### Screenshots
![image](https://user-images.githubusercontent.com/41262970/44978508-96266a80-af73-11e8-9a42-536cbc7bbe24.png)
